### PR TITLE
Fix email column inconsistencies in members list

### LINKED ISSUE #1164: [C-1939] More than two emails after member enrichment make the emails column Inconsistent
Following member enrichment (see screenshot), the 'emails' column overflowed with multiple entries, causing UI inconsistencies. Instead, we could display '+2' or similar to represent additional emails, akin to our handling of excess tags.

<sub>From [SyncLinear.com](https://synclinear.com) | [C-1939](https://linear.app/crowddotdev/issue/C-1939/more-than-two-emails-after-member-enrichment-make-the-emails-column)</sub>

### PR DESCRIPTION
**Pull request by peoray**
# Changes proposed ✍️

### What
Fixes #1164

### <samp>🤖 Generated by Copilot at dd328ab</samp>

Improved the UI and functionality of the member list table component in `member-list-table.vue`. Fixed a bug with an import statement.
​

### <samp>🤖 Generated by Copilot at dd328ab</samp>

> _We're haulin' on the member list, me hearties, yo ho ho_
> _We're limitin' the email badges, to make the table flow_
> _We're poppin' up a popover, with mailto links and more_
> _We're usin' `storeToRefs`, to access the store_

### <samp>🤖 Generated by Copilot at dd328ab</samp>

*  Limit and show email badges in member list table ([link](https://github.com/CrowdDotDev/crowd.dev/pull/1479/files?diff=unified&w=0#diff-4440397ab723cae9eaefae96077cf8e899c2f9feebd23c4d430d9f5d5d054b18L316-R316), [link](https://github.com/CrowdDotDev/crowd.dev/pull/1479/files?diff=unified&w=0#diff-4440397ab723cae9eaefae96077cf8e899c2f9feebd23c4d430d9f5d5d054b18R339-R377))
* Remove duplicate import of `storeToRefs` function ([link](https://github.com/CrowdDotDev/crowd.dev/pull/1479/files?diff=unified&w=0#diff-4440397ab723cae9eaefae96077cf8e899c2f9feebd23c4d430d9f5d5d054b18L414))

## Checklist ✅
- [ ] Label appropriately with `Feature`, `Improvement`, or `Bug`.
- [ ] Add screehshots to the PR description for relevant FE changes
- [ ] New backend functionality has been unit-tested.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [ ] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.

### CODE REVIEW

**Comment by peoray** [comment]
@yeganathan18 please take a look :)

**Comment by yeganathan18** [comment]
@peoray Looks good to me, It would be great if you could attach a screenshot, just to ensure everything is good 🙂

File: frontend/src/modules/member/components/list/member-list-table.vue [hunk 1]
```diff
@@ -336,6 +336,45 @@
...
+                        popper-class="support-popover"
+                      >
+                        <template #reference>
+                          <span
+                            class="badge--interactive hover:text-gray-900"
+                          >+{{ scope.row.emails.length - 3 }}</span>
+                        </template>
+                        <div class="flex flex-wrap gap-3 my-1">
+                          <el-tooltip
+                            v-for="email of scope.row.emails"
```
Comment (joanagmaia) [inline frontend/src/modules/member/components/list/member-list-table.vue:353]: I think Nuno's image might have induced an error, but the goal inside the popover is that it only displays the emails that are not visible on this list.

So let's say you have 5 emails. On the list it should display 3 and on the popover it should show the remaining 2 (that's why there's the +2 tag to indicate that 😄 )

Screenshot 2023-09-13 at 16 19 54
Screenshot 2023-09-13 at 16 19 57

**Review by joanagmaia — CHANGES_REQUESTED** [review]

File: frontend/src/modules/member/components/list/member-list-table.vue (hunk 1 above)
Comment (peoray) [reply frontend/src/modules/member/components/list/member-list-table.vue:353]: got that 👍🏽 , will make the changes

Comment (peoray) [reply frontend/src/modules/member/components/list/member-list-table.vue]: fixed

**Review by joanagmaia — APPROVED** [review]
LGTM 🚀

**Pull request merged**